tools/restatectl/src/commands/snapshot/create_snapshot.rs

```diff
@@ -99,8 +99,9 @@ async fn inner_create_snapshot(
         .into_inner();
 
     c_println!(
-        "Snapshot created for partition {partition_id}: {}",
-        response.snapshot_id
+        "Snapshot created for partition {partition_id}: {} (LSN >= {})",
+        response.snapshot_id,
+        response.min_applied_lsn,
```

Comment:
As a habbit, I wish we communicate the log-id when we talk about LSNs in general. I understand that the current equivalence of partition-id to log-id makes it redundant,, but it'll not always be like that.